integration_tests/support/commands.ts

```diff
@@ -1,4 +1,34 @@
+import RequestLinkPage from '../pages/link/requestLink'
+import featureFlags from './featureFlags'
+import RequestOneTimeCodePage from '../pages/one-time-code/requestOneTimeCode'
+import EmailSentPage from '../pages/one-time-code/emailSent'
+import Page from '../pages/page'
+
 Cypress.Commands.add('signIn', (options = { failOnStatusCode: true }) => {
   cy.request(`/`)
   cy.task('getSignInUrl').then((url: string) => cy.visit(url, options))
 })
+
+Cypress.Commands.add('signInAsLegalSender', () => {
```

Comment:
This is the implementation of the `signInAsLegalSender` abstraction. Its based around the value of the feature flag and either signs in using magic link or OTC